code/processes/dqe.q

```diff
@@ -113,8 +113,24 @@ postback:{[runtype;idnum;proc;params;result]
     .dqe.updresultstab[runtype;idnum;.z.p;first result;result[1];`complete;params;proc]];
   }
 
+containerfn:{[funct;names;vars]
```

Comment:
I don't really understand the goal this function, can you add some more comments etc.? is it just to make copies of all hdb tables with most recent partition or something?

Reply:
This function is to check if the passed over function is being run on a table in a HDB. Then if it is, a where clause will be added to stop a full HDB table being called.
I will add more comments to make it clear :)